utilities_export.py

```
@@ -1,3 +1,547 @@
+                            #print(file)
+                            #print(file, 'x', file_on_pdf, 'y', rect)
+                            mediay = (r.y1 + r.y0) / 2.0    
+                            if(p1y >= mediay and p0y <= mediay and (file, pathdocespecial) not in imagens):
+                                if("#" in file):
+                                    continue
+                                
+                                file = utilities_general.get_normalized_path(file)
+                                filepath = str(Path(utilities_general.get_normalized_path\
+                                                    (os.path.join(Path(utilities_general.get_normalized_path(pathdocespecial)).parent,file))))  
+                                if(filepath in japegos):
+                                    if(not (filepath, '') in japegos2):
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)